package.json

```diff
@@ -18,6 +18,7 @@
     "@angular/platform-browser-dynamic": "~2.1.0",
     "@angular/router": "~3.1.0",
     "core-js": "^2.4.1",
+    "devextreme": "^16.2.2",
```

Comment:
It`s better to use 16.2-unstable tag until devextreme stable version is not released.